tests/wpt/meta/content-security-policy/form-action/form-action-src-default-ignored.sub.html.ini

```diff
@@ -0,0 +1,3 @@
+[form-action-src-default-ignored.sub.html]
+  [Expecting logs: ["PASS","TEST COMPLETE"\]]
```

Comment:
Such test expectations are always confusing.

Can we upload another PR to rename the test cases first?

Reply:
This happens because the message generated by the log assertion script was too simple -- https://github.com/servo/servo/blob/b57428266710be3e646e43119de89e9d6a916ca5/tests/wpt/tests/content-security-policy/support/logTest.sub.js.

I am in favor to change the script and the message, but I don't think this PR should depend on that.